lib/extensions.js

```diff
@@ -252,6 +256,10 @@ module.exports = function (realmConstructor) {
                 if (cancelled) {
                   return;
                 }
+                // Clear the fallback timeOut if it has been started
+                if (timeoutId) {
+                  clearTimeout(timeoutId);
+                }
```

Comment:
Calling this with `null` is actually okay: https://developer.mozilla.org/en-US/docs/Web/API/clearTimeout#notes

```suggestion
                clearTimeout(timeoutId);
```